.gitignore

```diff
@@ -8,4 +8,5 @@ bin/
 build/
 
 fleet-server
+fleet_server
```

Comment:
When is this created?

Reply:
By my IDE, that won't let me remain it. ;-)